### PR TITLE
Remove unnecessary steps in the relational DB lesson #1380

### LINKED ISSUE #1380: Remove unnecessary steps in our relational DB lesson
Split from #1202 

The relational DB lesson is far too lengthy. It should introduce some basics, ensure the participant has installed and can successfully use Postgres, and leave the rest of it up to the other lessons, e.g. [SQL Part 1](https://github.com/Techtonica/curriculum/blob/master/databases/sql-1.md)

Edit this file:
[/databases/relational-databases.md](https://github.com/Techtonica/curriculum/blob/master/databases/relational-databases.md)

### Items to Remove
- [x] Remove `difference of MySQL vs PSQL` article - as we're not going to use MySQL this is a useless article
- [x] remove example companies and replace with an infographic about use of SQL vs NoSQL databases
- [x] SQL dialects
- [x] ORM's
- [x] Modeling data (covered in other topics)
- [x] Remove LucidChart database example and links
- [x] Remove "Users of databases" - it's also a confusing terminology
- [x] Replace "verb" with "operator" for the query actions - "verb" is not standard terminology
- [x] remove LIMIT examples
- [x] remove JOIN - that's a later topic
- [x] remove the second type of INSERT (where column order is named)
- [x] Remove DDL section
- [x] remove "advantages of relational" section
- [x] foreign key constraints
- [x] Remove all of the mistakes / misconceptions

### PR DESCRIPTION
**Pull request by iamdhruvsharma**
I have completed all the items in the checklist except "**Remove LucidChart database example and links**" as I was not sure about it. So, I request you to review this and let me know about the LucidChart.

Closes #1380

### CODE REVIEW

File: databases/relational-databases.md [hunk 1]
```diff
@@ -37,35 +34,27 @@ To understand, query, and insert information into a relational database, a techn
 - Explain the concept of a relational database and the use of schemas to enforce structure
 - Execute basic SQL commands for CRUD with a database
 - Perform basic selects and joins to find the requested information
```
Comment (alodahl) [inline databases/relational-databases.md:36]: Can you remove all references to "join"? They haven't learned it yet in the curriculum.

Comment (iamdhruvsharma) [reply databases/relational-databases.md:36]: Sure! on it.

Comment (iamdhruvsharma) [reply databases/relational-databases.md]: Done!

**Comment by iamdhruvsharma** [comment]
Thank you for your review. Yes, I would like to help with the issue #1202

**Pull request merged**